### ISSUE TITLE
NhUsers group dead?

### ISSUE THREAD

**Issue by fredericDelaporte** [nhibernate/nhibernate-core#2532]
It seems posts are no more approved on the [NhUsers group](https://groups.google.com/g/nhusers).

I have posted all v5.3.x release announcements there, but none have been approved. I do not have the rights to do that.
If none of the current approvers are active anymore, the group is then dead. This is a trouble since we ask people to go there for support.

**Comment by bahusoid** [comment]
It seems GitHub Discussions  should soon go public https://github.com/github/roadmap/issues/104 (Q4 2020 – Oct-Dec). I think we should start using it instead of mailing list.

**Comment by hazzik** [comment]
Strange, I was seeing option "Discussion" with a badge "Preview", but now I don't see it.  Should have enabled it when it was available. Duh...

**Comment by oskarb** [comment]
I remember I looked into this back in 2015 and asked the group owner to make me an owner too (and some others), but got no reply. I now tried again on a different email address.

I am a manager so I can approve, but didn't pay attention to it because someone else always seemed to beat me to it. :)

**Comment by oskarb** [comment]
Well, Bil replied now and says he made me an owner, but I don't see it yet. It's confusing though because during the last couple of minutes, suddenly Frédéric appeared as owner.

**Comment by oskarb** [comment]
Right, so I sent a bad link to Bil which led him to make Frederic an owner, which explains the mystery. He now also added me, and I also promoted Alex so there are now three of us besides Bil.

This at least secures the current situation.

Frederic, I also edited you so that your posts are allowed without moderation.

To approve/reject, one does not need Owner, it is enough to be Manager. Of the current managers, I recognize them as people that used to be active but that I haven't really noticed in quite a while. Maybe we should add a few more?
